PlaybackLab/DownloadToOwnSampleApp/README.md

```diff
@@ -20,21 +20,19 @@ To run this app, we assume you have an Apple developer account because you need
-* **OOAssetDownloadManagerDelegate**: Build a class implementing this protocol to know about the progress of a download. Remember setting the class as a delegate of `OOAssetDownloadManager`.
+* **OODtoAsset**: Downloads a single video asset. Needs an `PlayerSelectionOption` to initialize. Use appropriate methods with closures `-downloadWithProgressClosure: -progressWithProgressClosure: -finishWithRelativePath: -onErrorWithErrorClosure:` to start downloading and to know about the progress of a download.
+* **PlayerSelectionOption**: Provides information to `OOAssetDownloadOptions` so it knows which asset to download.
+* **OOAssetDownloadOptions**: Provides information to `OODtoAsset` so it knows which asset to download.
+* **OOEmbedTokenGenerator**: An instance of this protocol is used to enable our Ooyala Player Token (OPT) protection feature. This is necessary for assets using entitlements and Fairplay DRM protected assets. When downloading a Fairplay asset, you must provide instance of `OOEmbedTokenGenerator` to `OOAssetDownloadOptions` before pass it to `OODtoAsset`.
 * **OOOoyalaPlayer**: The player we always provide to playback video.
 * **OOOoyalaPlayerViewController**: Used to manage the `view` of the player.
 
 On top of these classes, we built the provided sample app that you can find here. It uses all the functionality described in the objects above. Here are the important classes you want to check out:
-
-* **AssetPersistenceManager**: This is a singleton object which manages multiple `OOAssetDownloadManager` objects. It means you can check this example to manage multiple downloads at the same time. This class is the `OOAssetDownloadManagerDelegate` for all the `OOAssetDownloadManager` objects, so you'll get notified here on the progress of a download. `AssetPersistenceManager` broadcasts notifications in the app so anyone interested can add itself as an observer of these notifications and act accordingly as the downloads progress.
 * **AssetListViewController**: Manages the `UITableView` showing the assets of the app. Depending on the download state of each asset, it will show different options when tapping the accessory button of each cell. Options can be _Download_, _Cancel_, or _Delete_.
-* **OptionTableViewCell**: An instance of each cell rendered in the `UITableView`. Each cell is an observer of the download notifications sent by `AssetPersistenceManager`, so the cell refreshes the UI when there's a new download state.
+* **OptionTableViewCell**: An instance of each cell rendered in the `UITableView`. Each cell passes it's block with task to appropriate instance of `OODtoAsset` to be notified when download state changes, so the cell refreshes the UI.
```

Comment:
If you look at `OptionTableViewCell` you'll see that now it doesn't have any logic inside.

Reply:
I agree that not cell itself refresh UI, but handler in UIAlertAction. Handler updates cell's outlets by info from dtoAsset state. I moved description of UI-refreshing process from `OptionTableViewCell` to `AssetListViewController`